### PR TITLE
add a fancy new decorator to the upper right hand side of the podcast…

### PR DESCRIPTION
**Pull request by Rowansdabomb**
… buttons throughout the site

Looks like so in the header

image

### CODE REVIEW

**Comment by Rowansdabomb** [comment]
This PR also sets a smaller font size for the episode title, as it was not clearly different from the podcast title before

File: ubyssey/static/src/styles/main.scss [hunk 1]
```diff
@@ -88,6 +88,14 @@ body, html, #page-wrapper {
   -webkit-overflow-scrolling: touch;
 }
 
+span.upper-right-decorator {
+  color: #006DCC;
+  font-size: .8em;
+  position: relative;
+  top: -.8em;
+  font-family: arial;
```
Comment (psiemens) [inline ubyssey/static/src/styles/main.scss]: Can this be `$font-default`?

File: ubyssey/static/src/styles/main.scss [hunk 2]
```diff
@@ -88,6 +88,14 @@ body, html, #page-wrapper {
   -webkit-overflow-scrolling: touch;
 }
 
+span.upper-right-decorator {
+  color: #006DCC;
```
Comment (psiemens) [inline ubyssey/static/src/styles/main.scss]: This can be replaced with `$color-ub-blue-light`

**Review by psiemens — APPROVED** [review]
Nice!

File: ubyssey/static/src/styles/main.scss (hunk 2 above)
Comment (Rowansdabomb) [reply ubyssey/static/src/styles/main.scss]: yea

File: ubyssey/static/src/styles/main.scss (hunk 1 above)
Comment (Rowansdabomb) [reply ubyssey/static/src/styles/main.scss]: yes

File: ubyssey/static/src/styles/main.scss [hunk 3]
```diff
@@ -88,6 +88,14 @@ body, html, #page-wrapper {
   -webkit-overflow-scrolling: touch;
 }
 
+span.upper-right-decorator {
+  color: $color-ub-blue-light;
+  font-size: .8em;
```
Comment (jumbosushi) [inline ubyssey/static/src/styles/main.scss:93]: Might look better if we make it smaller for Mobile

image

File: ubyssey/templates/headers/topbar.html [hunk 4]
```diff
@@ -21,7 +21,7 @@
                 </li><li {% if section == 'events' %}class="current"{% endif %}>
                     <a href="{% url 'events' %}">Events</a>
                 </li><li {% if section == 'podcast' %}class="current"{% endif %}>
-                    <a href="{% url 'podcasts' 'episodes' %}">Podcast</a>
+                    <a href="{% url 'podcasts' 'episodes' %}">Podcast <span class='upper-right-decorator'>New</span></a>
```
Comment (jumbosushi) [inline ubyssey/templates/headers/topbar.html:24]: Can we also add a podcast link in a mobile header as described in https://github.com/ubyssey/ubyssey.ca/issues/593 ?

**Review by jumbosushi — CHANGES_REQUESTED** [review]
Looks great :100:

File: ubyssey/static/src/styles/main.scss (hunk 3 above)
Comment (Rowansdabomb) [reply ubyssey/static/src/styles/main.scss:93]: Playing around with this, I don't see much return for effort. These new buttons will likely be removed in a month or so.

**Pull request merged**